src/components/DropDownButton.tsx

```diff
@@ -184,6 +185,11 @@ class DropDownButton extends React.Component<
 						alignRight={alignRight}
 						onClick={this.toggle}
 						minWidth={`${this.state.minWidth}px`}
+						maxHeight={
+							isNumber(this.props.maxHeight)
```

Comment:
You can use the `px()` function from `utils.ts` here https://github.com/balena-io-modules/rendition/blob/master/src/utils.ts#L40

Reply:
Oh cool, thanks!

Reply:
@LucianBuzzo Done, have a look at it when you find the time.